app/models/concerns/rubygem_searchable.rb

```diff
@@ -37,9 +40,13 @@ def as_indexed_json(_options = {})
              }
 
     mapping do
-      indexes :name, analyzer: 'rubygem'
-      indexes :indexed, type: 'boolean'
-      indexes :info, analyzer: 'english'
+      indexes :name, type: 'multi_field' do
```

Comment:
The `multi_field` option is actually useless in current Elasticsearch — but I'll have to revisit the example application first, then we can carry it over.